### PR TITLE
Feature/prlc100 65 typeoforder

### PR DESCRIPTION
**Pull request by rahulsanjava19**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/app/form/validation.ts [hunk 1]
```diff
@@ -28,7 +28,14 @@ export const isFieldFilledIn: Validator = value => {
 };
 
 export const atLeastOneFieldIsChecked: Validator = fields => {
```
Comment (channa-patil) [inline src/main/app/form/validation.ts:30]: @rahulsanjava19 Will this not impact others using the function?
If yes, can we can add it as new function for c100 please?

File: src/main/steps/c100-rebuild/typeoforder/select-courtorder/content.ts [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+export const form: FormContent = {
+  fields: {
+    courtOrder: {
+      id: 'courtOrder',
+      type: 'checkboxes',
+      hint: l => l.select_all_apply,
+      validator: atLeastOneFieldIsChecked,
+      values: [
+        {
+          name: 'courtOrder',
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/typeoforder/select-courtorder/content.ts:97]: Hope we are saving the selected values into session, as we need to render next screens based on this selection.

**Review by channa-patil — CHANGES_REQUESTED** [review]

File: src/main/steps/c100-rebuild/typeoforder/select-courtorder/content.ts (hunk 2 above)
Comment (rahulsanjava19) [reply src/main/steps/c100-rebuild/typeoforder/select-courtorder/content.ts:97]: Yes, Framework does save in userCase automatically same as other screens

**Review by channa-patil — APPROVED** [review]

File: src/main/app/form/validation.ts (hunk 1 above)
Comment (rahulsanjava19) [reply src/main/app/form/validation.ts:30]: we filtered values to remove empty values so it won't break. Existing test cases passed as well where this functions is used

**Pull request merged**